tests/functional/intelligent-host-selection/03-polling/flow.cylc

```diff
@@ -1,48 +1,49 @@
 [meta]
 title = "Try out scenarios for intelligent host selection."
 description = """
-Tasks
-- goodhost: a control to check that everything works
-- mixedhost contains some hosts that will and won't fail
+Runs two long running tasks on a known good platform and on a
+platform with some unreachable hosts.
+
+Once each has started trigger a task which kills each to test the
+execution polling.
 """
 
 [scheduler]
     [[events]]
-        expected task failures = 1/t1, 1/t2
+        expected task failures = 1/goodhosttask 1/mixedhosttask
 
 
```

Comment:
Ah, one more small thing, but probably best to address to avoid confusion later down the line: this is supposed to be comma separate, but seeing as the test doesn't do a reftest, it's actually ignored
```suggestion
```

Reply:
@MetRonnie  can you check you are happy with my change and merge. (I didn't use your suggestion directly because it GH wasn't playing ball, and if left rather a lot of whitespace)